interfaces/policy/basedeclaration_test.go

```diff
@@ -622,6 +622,7 @@ var (
 		"pulseaudio":  {"app", "core"},
 		"serial-port": {"core", "gadget"},
 		"spi":         {"core", "gadget"},
+		"uio":         {"core", "gadget"},
```

Comment:
Since other changes are needed, can you put this in alphabetically? (I know dummy is the outlier, but `// other` is meant to be alphabetized.

Reply:
I fought with this file because of gofmt breaking the diff in my editor. In the end I used nano to add this line, hoping I won't have to touch it again.

The problem is gofmt on my system is too new for our static checker.

Reply:
Done, I hope.